userguide/kaui/includes/users.adoc

```diff
@@ -0,0 +1,165 @@
++
+image::Homepage-SettingsDropdown.png[width=650,align="center"]
++
+. Select *Users.*
++
+Kaui displays [SOMETHING]. _I don't have access to this screen._
++
+. Select [SOMETHING ELSE].
++
+Kaui displays the Add New User pop-up:
++
+image::AddNewUser.png[width=450,align="center"]
++
+[NOTE]
+*Note:* You can also open the Add New User screen by clicking on the username and tenant name in the upper right corner and clicking the plus icon ( image:i_PlusGreen.png[] ) next to "Allowed Users."
++
+. Fill in the fields. For field descriptions, see <<Add New User Field Descriptions>>.
+. Click the *Save* button.
+
+. CONFIGURE TENANT STEP HERE?
+
+image::ConfiguredTenants.png[width=650,align="center"]
++
+. Select the tenant(s) the user has access to.
+. Click the *Save* button.
+
+== Edit a User
+
+Use the steps in this section to change the user's name, password, or role.
+
+To edit a user:
+
+. Click on your username and tenant name in the upper right corner: <--CAN ALSO GO TO SETTINGS->USERS, BUT I DON'T HAVE ACCESS
++
+image::Users-SelectTenantName.png[width=650,align="center"]
++
+. In the list of Allowed Users, click on the user to edit:
++
+image::AllowedUsersList.png[width=650,align="center"]
++
+^^SAME COMMENT AS ABOVE ABOUT NEEDING A SCREENSHOT WITH A LONGER USER LIST
++
+Kaui displays the User Details screen.
++
+image::UserDetails.png[width=450,align="center"]
++
+. Click *Edit* (next to "User Details").
++
+Kaui displays the Update User screen:
++
+image::UpdateUser.png[width=450,align="center"]
++
+. Fill in the user fields. For field descriptions, see <<Add New User Field Descriptions>>.
+. Click the *Save* button.
+
+== Add New User Field Descriptions
+
+[cols="1,3"]
+|===
+^|Field ^|Description
+
+| Managed externally (LDAP, Okta, etc.)?
+| Check this box if you are managing users in a third-party system. Otherwise, leave it unchecked.
+
+| Name
+| Enter the user's name. _QST: Name should be first, last, both? Is email address here optional?_
+
+| Password
+| Enter the user's password. For security, Kaui does not display any existing passwords in this field, but you can type in a different one. It may be easier to copy and paste the password into this field.
+
+| Description
+| To provide a brief description of this user or his/her role on the Allowed Users list, type it here.
+
+| Roles
+| Enter the roles that are associated with this user. You also have the option to <<Create a New Role, create a new role>>.
+
+|===
+
+== Delete a User
+
+. Click on your username and tenant name in the upper right corner:
++
+image::Users-SelectTenantName.png[width=650,align="center"]
++
+. In the list of Allowed Users, click on the user to edit:
++
+image::AllowedUsersList.png[width=650,align="center"]
++
+Kaui displays the User Detail screen.
++
+image::UserDetails.png[width=450,align="center"]
++
+. Click the image:i_RedX.png[] icon. Kaui displays a message to confirm the deletion.
++
+. Click *OK*.
```

Comment:
AFAIK, these steps only remove the user from the "allowed users" list for the current tenant. In order to permanently delete the user from the system, you need to perform delete from the `Settings -> Users` screen  (where the user is still available even after deleting from the "allowed users" screen).  But I'm not sure, so please check with @pierre or @sbrossie

Reply:
I think it actually deletes the user, that's the implementation:

https://github.com/killbill/killbill-admin-ui/blob/master/app/models/kaui/allowed_user.rb#L57
https://github.com/killbill/killbill-client-ruby/blob/44cd82a622fb6a453f71211b6b38a5edbd459653/lib/killbill_client/models/user_roles.rb#L31

Are you seeing something different?

Reply:
Yes, deleting the user from the `Tenant` screen (displayed on clicking username and tenant name in the upper right corner) only removes the user from the `allowed users` list. The user is still visible on clicking `SETTINGS > USERS`.  If this is not the expected behavior, I will create a task to investigate this further.

Reply:
Oh, I suspect the Users screen still sees the "Kaui user" (Kaui has its own view of users), while the other screen performs the deletion in Kill Bill.

Yes, let's create a task to investigate further 👍 Thanks!